src/index.js

```diff
@@ -1,29 +1,95 @@
 import './style.css';
 
-const todoListArr = [
-  {
-    description: 'Take the car to the mechanic',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Attend choir practice at 6:00 PM',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Do some coding challenges on data structures',
-    completed: false,
-    index: 3,
-  },
-];
-
-todoListArr.forEach((todo) => {
-  const todoContent = `
-  <div>
-  <input type="checkbox" class=checkbox>
-  <span>${todo.description}</span>
+const todoListSection = document.getElementById('todo-list');
+const todoText = document.querySelector('input');
+
+class TodoObject {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const removeTodoList = (todo) => {
+  todoListSection.removeChild(todo);
+  const localdata = JSON.parse(localStorage.getItem('todolist'));
+  const data = Array.from(localdata).filter((i) => i.completed === false);
+  data.map((i) => i.index);
+  localStorage.setItem('todolist', JSON.stringify(data));
+};
```

Comment:
- Currently, when I delete an item it is not removed from the array in the local storage. It disappears from the page but the data remain intact in the storage. Please look into this and make sure that the storage is updated on every task deleted.
    * Remember: deleting a task should also update all remaining items' indexes, so they represent the current list order.

![image](https://user-images.githubusercontent.com/38088818/173178597-8afd8274-78ad-47de-968b-d4f4f9bb872b.png)

- Similarly, when I reload the page all tasks disappear on the page yet the data still exists in the local storage. Please implement a function that will make sure that data is loaded from the local storage on page load. 

![image](https://user-images.githubusercontent.com/38088818/173178435-5b8d50c3-8133-4ecc-832e-6f68d3f54a64.png)